Sources/Sender.swift

```diff
@@ -43,3 +43,10 @@ extension Sender: Equatable {
         return left.id == right.id
     }
 }
+
+// MARK: - Hashable Conformance
+extension Sender: Hashable {
```

Comment:
This is probably a bit premature but not a bad idea. I'd probably take things a bit further by XORing all the underlying hash values.